client/blocks/feedback/edit.js

```diff
@@ -200,6 +201,12 @@ const EditFeedbackBlock = ( props ) => {
 		height,
 	};
 
+	const isCurrentlyClosed =
```

Comment:
Nit: Not a huge fan of overly descriptive variable names, let's maybe cut it down to `isClosed`?